### PR TITLE
static loader for checks + checks_paths: always use the default checks

### PR DESCRIPTION
**Pull request by TomasTomecek**
This will make things slightly more easier when developing local, external checks.

### CODE REVIEW

**Comment by TomasTomecek** [comment]
no longer a minor thing

I may try to fix them if they are not broken badly

File: colin/core/loader.py [hunk 1]
```diff
@@ -124,6 +124,24 @@ def obtain_check_classes(self):
...
         return list(check_classes)
 
+    def import_class(self, import_name):
+        """
+        import selected class
+
+        :param import_name, str, e.g. some.module.MyClass
+        :return the class
+        """
+        if six.PY3:
```
Comment (lachmanfrantisek) [inline colin/core/loader.py]: Probably unnecessary. From the last release, we do not support Python2.

Comment (TomasTomecek) [reply colin/core/loader.py]: nice! shall I remove all the python2 junk from this file?

**Comment by TomasTomecek** [comment]
I'll try to take a look at the tests, shouldn't be that hard to fix them

File: colin/core/loader.py (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/core/loader.py]: A hope so.

**Comment by TomasTomecek** [comment]
I hope that I'm done here, we shall see about them tests, they passed locally for me.

`podman mount` doesn't work with ovl + rootless, so we need to run them as root

**Comment by TomasTomecek** [comment]
`=== 5 failed, 89 passed, 1 error in 390.89 seconds ===`

They are passing for me locally. No idea how to make the CI failures pass.

**Comment by TomasTomecek** [comment]
`==== 94 passed in 129.33 seconds ====` locally and it should also fix CI runs

**Comment by lachmanfrantisek** [comment]
@TomasTomecek Thanks, the CI works!

**Pull request merged**